package.json

```diff
@@ -132,7 +132,7 @@
     "resin-image-fs": "^2.3.0",
     "resin-image-manager": "^5.0.0",
     "resin-preload": "^5.0.0",
-    "resin-sdk": "^7.0.0",
+    "resin-sdk": "^8.1.0",
```

Comment:
This is a surprising change to include here! Why is this in this PR? Does this definitely not need other changes anywhere?

Reply:
It contains the new getDashboardUrl for sure.

Reply:
@pimterry , if I didn't change the resin-sdk version I had errors in the build, but you are right we need a new pull request for that, checking all the changes in the new sdk version. I will try to check which 7.x.x version I should use